common/mobilization_options/00_mobilization_option.txt

```diff
@@ -0,0 +1,295 @@
+		unit_recovery_rate_add = 0.05	#0
+		unit_offense_mult = -0.05	#0
+		unit_defense_mult = -0.05	#0
+	}
+	ai_value = 1
+
+	group = supplements
+}
+
+mobilization_option_forced_march = {
+	texture = "gfx/interface/icons/production_method_icons/skirmish_infantry.dds"
+	upkeep_modifier = {
+		# empty on purpose - this has no cost and is just a toggle with its own tradeoffs
+	}
+	unit_modifier = {
+		military_formation_travel_speed_mult = 0.1
+		unit_mobilization_speed_mult = 0.25
+		unit_morale_loss_mult = 0.1
+	}
+	ai_value = 1
+
+	group = transport
+}
+
+mobilization_option_truck_transport = {
+	texture = "gfx/interface/icons/production_method_icons/public_motor_carriages.dds"
+	unlocking_technologies = {
+		combustion_engine
+	}
+	upkeep_modifier = {
+		goods_input_transportation_add = 0.5
+		goods_input_automobiles_add = 0.5
+	}
+	unit_modifier = {
+		military_formation_travel_speed_mult = 0.2
+		unit_mobilization_speed_mult = 0.4
+	}
+	ai_value = 1
+
+    group = transport
+}
+
+mobilization_option_rail_transport = {
+	texture = "gfx/interface/icons/production_method_icons/rail_transport.dds"
+	unlocking_technologies = {
+		railways
+	}
+	upkeep_modifier = {
+		goods_input_transportation_add = 2
+	}
+	unit_modifier = {
+		military_formation_travel_speed_mult = 0.3
+		unit_mobilization_speed_from_railways_mult = 0.5
+	}
+	ai_value = 1
+
+    group = transport
+}
+
+mobilization_option_machinegunners = {
+	texture = "gfx/interface/icons/production_method_icons/unused/powered_machine_guns.dds"
+	unlocking_technologies = {
+		automatic_machine_guns
+	}
+	upkeep_modifier = {
+		goods_input_small_arms_add = 1
+		goods_input_ammunition_add = 2	#1
+	}
+	unit_modifier = {
+		unit_offense_add = 5
+		unit_defense_add = 10
+	}
+	ai_value = 1
+
+    group = special_weapons
+}
+
+mobilization_option_chemical_weapons = {
+	texture = "gfx/interface/icons/production_method_icons/chemical_weapons_specialists.dds"
+	unlocking_technologies = {
+		chemical_warfare
+	}
+	upkeep_modifier = {
+		goods_input_fertilizer_add = 4	#2
+	}
+	unit_modifier = {
+		unit_offense_add = 20
+		unit_kill_rate_add = 0.5
+		unit_recovery_rate_add = -0.2
+	}
+	ai_value = 1
+
+    group = special_weapons
+}
+
+mobilization_option_flamethrowers = {
+	texture = "gfx/interface/icons/production_method_icons/flamethrower_company.dds"
+	unlocking_technologies = {
+		flamethrowers
+	}
+	upkeep_modifier = {
+		goods_input_oil_add = 1
+	}
+	unit_modifier = {
+		unit_offense_add = 15
+		unit_devastation_mult = 0.5
+		unit_morale_damage_mult = 0.1
+	}
+	ai_value = 1
+
+    group = special_weapons
+}
+
+# mobilization_option_landing_crafts = {
+# 	texture = "gfx/interface/icons/production_method_icons/flamethrower_company.dds"
+# 	unlocking_technologies = {
+# 		landing_craft
+# 	}
+# 	upkeep_modifier = {
+# 		goods_input_ironclads_add = 1
+# 	}
+# 	unit_modifier = {
+# 		#needs code support to reduce naval invasion penalty
+# 	}
+# 	ai_value = 1
+
+#     group = special_weapons
+# }
```

Comment:
I assume this is vanilla commented out code? xd

Reply:
Yes